specification/containerservice/Fleet.Management/fleetnamespace.tsp

```diff
@@ -34,47 +36,54 @@ model FleetManagedNamespace
   ...Azure.ResourceManager.Foundations.ArmTagsProperty;
```

Comment:
please check @key and @segment above - should not have the fleet prefix IMO

Reply:
Apologies for lack of clarity, I think the model names should be prefixed with Fleet, but I think that key and segment maybe shouldn't be? (see FleetUpdateStrategy)

Reply:
Using FleetManagedNamespaceProperties at the top level enables us to use ManagedNamespaceProperties for the enclosed struct, which maybe helps us too